src/main/java/use_cases/display_task_tree/TaskTreeUIInterface.java

```diff
@@ -0,0 +1,4 @@
+package use_cases.display_task_tree;
+
+public interface TaskTreeUIInterface {
```

Comment:
Make sure they you end up having your TaskTreeUI implement this interface! This is going to be the key in presenting your TaskTreeDisplayModel to the UI.